lib/chromedriver.js

```diff
@@ -256,22 +269,36 @@ class Chromedriver extends events.EventEmitter {
     return chromeVersion ? semver.coerce(chromeVersion) : null;
   }
 
+  /**
+   *
+   * @param {ChromedriverVersionMapping} newMapping
+   * @returns {Promise<void>}
+   */
   async updateDriversMapping (newMapping) {
     let shouldUpdateStaticMapping = true;
+    if (!this.mappingPath) {
```

Comment:
if this.mappingPath is wrong fs.exists should return false, which is expected. We don't have to return here

Reply:
we do because `fs.exists()` expects a `string` parameter, and we don't have one, so it can't be called.